source/game/g_ascript.cpp

```diff
@@ -656,18 +652,23 @@ static const asClassDescriptor_t asTeamListClassDescriptor =
 };
 
 // CLASS: Stats
-static void objectScoreStats_Clear( score_stats_t *obj ) {
-	memset( obj, 0, sizeof( *obj ) );
+static void objectScoreStats_AddScore( int add, score_stats_t *obj ) {
+	obj->score += add;
 }
 
-static void objectScoreStats_ScoreSet( int newscore, score_stats_t *obj ) {
-	obj->score = newscore;
+static void objectScoreStats_SetScore( int score, score_stats_t *obj ) {
+	obj->score = score;
 }
 
-static void objectScoreStats_ScoreAdd( int score, score_stats_t *obj ) {
-	obj->score += score;
+static void objectScoreStats_SetState( bool state, score_stats_t * obj ) {
+	obj->state = state;
```

Comment:
Can you rename state to has_bomb or something

Reply:
no because it includes ready